lib/node_modules/@stdlib/stats/base/nanmskmax/test/test.ndarray.js

```diff
@@ -206,16 +343,56 @@ tape( 'the function supports negative `stride` parameters', function test( t ) {
 		0
 	];
 
-	N = floor( x.length / 2 );
-	v = nanmskmax( N, x, -2, 10, mask, -2, 10 );
+	x = toAccessorArray( x );
+
+	v = nanmskmax( 6, x, -2, 10, toAccessorArray( mask ), -2, 10 );
+
+	t.strictEqual( v, 4.0, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function supports `offset` parameters', function test( t ) {
+	var mask;
+	var x;
+	var v;
 
+	x = [
+		2.0,
+		1.0,  // 0
+		2.0,
+		-2.0, // 1
+		-2.0,
+		2.0,  // 2
+		3.0,
+		4.0,  // 3
+		5.0,
+		6.0,  // 4
+		NaN,
+		NaN   // 5
+	];
+	mask = [
+		0,
+		0, // 0
+		0,
+		0, // 1
+		0,
+		0, // 2
+		0,
+		0, // 3
+		1,
+		1, // 4
+		0,
+		0  // 5
+	];
+
+	v = nanmskmax( 6, x, 2, 1, mask, 2, 1 );
 	t.strictEqual( v, 4.0, 'returns expected value' );
+
 	t.end();
 });
 
 tape( 'the function supports `offset` parameters', function test( t ) {
```

Comment:
```suggestion
tape( 'the function supports `offset` parameters (accessors)', function test( t ) {
```

kindly fix all other errors similar to this in the tests, it makes it hard for someone else to go through these if the descriptions are not accurate